providers/hns/hns_roce_u_verbs.c

```diff
@@ -175,6 +175,77 @@ int hns_roce_u_dereg_mr(struct verbs_mr *vmr)
 	return ret;
 }
 
+int hns_roce_u_bind_mw(struct ibv_qp *qp, struct ibv_mw *mw,
+		       struct ibv_mw_bind *mw_bind)
+{
+	struct ibv_mw_bind_info *bind_info = &mw_bind->bind_info;
+	struct ibv_send_wr *bad_wr = NULL;
+	struct ibv_send_wr wr = {};
+	int ret;
+
+	if ((mw->pd != qp->pd) || (mw->pd != bind_info->mr->pd))
+		return EINVAL;
+
+	if (mw->type == IBV_MW_TYPE_2)
+		return EINVAL;
+
+	if (!bind_info->mr && bind_info->length)
+		return EINVAL;
+
+	if (bind_info->mw_access_flags & IBV_ACCESS_ZERO_BASED)
```

Comment:
This should list the flags this code does support, not the flags it doesn't support.

Reply:
This code is used to do correctness checking, hip08 only support VA, not ZBVA. Should I keep this check or just delete it?